processes/ingest/doab.py

```diff
@@ -1,19 +1,26 @@
 import os
 from services import DSpaceService
-from ..core import CoreProcess
 from logger import create_log
 from mappings.doab import DOABMapping
-from managers import DOABLinkManager, S3Manager, RabbitMQManager
+from managers import DBManager, DOABLinkManager, S3Manager, RabbitMQManager
 from model import get_file_message
+from .record_buffer import RecordBuffer
 
 logger = create_log(__name__)
 
-class DOABProcess(CoreProcess):
+class DOABProcess():
     DOAB_BASE_URL = 'https://directory.doabooks.org/oai/request?'
     DOAB_IDENTIFIER = 'oai:directory.doabooks.org'
 
     def __init__(self, *args):
-        super(DOABProcess, self).__init__(*args[:4])
+        self.process = args[0]
```

Comment:
Not sure if this is the best way to get the arguments. Let me know if you know of a better option.

Reply:
For right now that is the best and easiest way to get the arguments.